lib/tasks/transifex.rake

```diff
@@ -49,6 +49,28 @@ namespace :transifex do
     end
   end
 
+  # Download translations from Transifex - tipline resource
+
+  task download_tipline: [:environment, :login] do
+    project = Transifex::Project.new('check-tiplines')
+    resource_slugs = project.resources.fetch.select{ |r| r['slug'] =~ /^hardcoded-bot-strings/ }.collect{ |r| r['slug'] }
```

Comment:
There is only one resource, right? So we could replace this `select` by a `find`.

Reply:
Even better, since the resource name is known we can just pass it to the `resource = project.resource(slug)` line